crypten/nn/module.py

```diff
@@ -130,17 +170,35 @@ def set_parameter_from_shares(self, name, share, **kwargs):
         setattr(self, name, self._parameters[name])
 
     def parameters(self, recurse=True):
-        """Iterator over parameters."""
+        """Returns an iterator over module parameters.
+        This is typically passed to an optimizer.
+        Args:
+            recurse (bool): if True, then yields parameters of this module
+                and all submodules. Otherwise, yields only parameters that
+                are direct members of this module.
+        Yields:
+            CrypTensor or torch.Tensor: module parameter
+        """
         for _, param in self.named_parameters(recurse=recurse):
             yield param
 
     def named_parameters(self, recurse=True, prefix=None):
```

Comment:
To make this consistent with `named_modules` (and PyTorch), maybe you can make `prefix=""` the default here?

Reply:
Sure. I set it to `None` since that is how CrypTen currently assigns.